etcd-manager/go.mod

```
@@ -2,31 +2,33 @@ module sigs.k8s.io/etcdadm/etcd-manager
 
 go 1.18
 
+replace k8s.io/client-go => k8s.io/client-go v0.24.2
+
 require (
 	cloud.google.com/go/compute v1.7.0
-	github.com/Azure/azure-sdk-for-go v65.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go v66.0.0+incompatible
 	github.com/Azure/go-autorest/autorest/azure/auth v0.5.11
 	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/aws/aws-sdk-go v1.44.56
 	github.com/blang/semver/v4 v4.0.0
 	github.com/digitalocean/godo v1.81.0
 	github.com/golang/protobuf v1.5.2
 	github.com/gophercloud/gophercloud v0.25.0
-	github.com/hetznercloud/hcloud-go v1.34.0
+	github.com/hetznercloud/hcloud-go v1.35.0
 	github.com/prometheus/client_golang v1.12.2
 	go.etcd.io/etcd/api/v3 v3.5.4
 	go.etcd.io/etcd/client/v3 v3.5.4
-	golang.org/x/net v0.0.0-20220615171555-694bf12d69de
-	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
-	google.golang.org/api v0.84.0
+	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
+	golang.org/x/oauth2 v0.0.0-20220628200809-02e64fa58f26
+	google.golang.org/api v0.86.0
 	google.golang.org/grpc v1.47.0
 	gopkg.in/gcfg.v1 v1.2.3
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
-	k8s.io/klog/v2 v2.60.1
-	k8s.io/kops v1.24.0-beta.2
-	k8s.io/mount-utils v0.24.2
-	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
+	k8s.io/apimachinery v0.24.3
+	k8s.io/client-go v1.5.2
```

Any idea why this changed to v1.5.2 and why replace was needed?
```suggestion
	k8s.io/client-go v1.5.2
```

---

I don't really understand any of these client-go changes that are required. It seems to come from replace being removed in kOps. And why 1.5.2? no idea. kubetest2-kops pulls in version v11. Nothing I do makes it stick with the proper version.

---

Solution seems to be to switch to `k8s.io/kops v1.25.0-alpha.1`, or do another 1.24 release.

---

Will wait for our next release then.